Jenkinsfile

```diff
@@ -142,43 +146,45 @@ pipeline {
 
         stage('Publish to S3') {
             agent {
-                label "linux && !mr-dl16"
+                label "docker"
```

Comment:
What is this doing now?

Reply:
it's using the dockerfile from the repo, which has s3cmd configured inside it already 

the archive and upload step should now run inside the container and upload to s3; this is the main purpose of this PR

Reply:
Makes sense and a good way to go.